src/views/Proposal.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <Layout>
+  <Layout v-bind="$attrs">
```

Comment:
Curious what this is doing? 🤔

Reply:
This is needed because we have multiple root nodes in the template. We where getting some warnings because of this.

https://v3.vuejs.org/guide/component-attrs.html#attribute-inheritance-on-multiple-root-nodes

We could also move the teleport into the layout as an alternative.